views/index.jade

```diff
@@ -0,0 +1,127 @@
+extend layout
+
+block scenarios
+
+  //- Left join each scenario with its sessions. Select only launchable scenarios (with documents)
+  //- and joinable sessions (with instances).
+
+  -
+    var application = "/ITDG/index.vwf",
+      scenarios = manifest[ application ] || [];
+    var scenario_sessions = _.sortBy( scenarios, function( scenario, name ) {
+      return name;
+    } ).filter( function( scenario ) {
+      return scenario.scenario.document;
+    } ).reduce( function( scenario_sessions, scenario ) {
+      var sessions = ( scenario.sessions || [] ).filter( function( session ) {
+        return session.instance;
+      } );
+      return scenario_sessions.concat( sessions.map( function( session ) {
+        return { scenario: scenario.scenario, session: session };
+      } ) ).concat( { scenario: scenario.scenario, session: undefined } );
+    }, [] );
```

Comment:
I moved these to a separate file. There isn't a built-in way to do this with express, but it can be done with just a little manual work.